src/elm/Vela.elm

```diff
@@ -1549,22 +1550,34 @@ type alias Log =
     , repository_id : Int
     , rawData : String
     , decodedLogs : String
+    , size : Int
     }
 
 
 {-| decodeLog : decodes json from vela into log
 -}
 decodeLog : Decoder Log
 decodeLog =
-    Decode.succeed Log
+    Decode.succeed
+        (\id step_id service_id build_id repository_id data ->
+            Log
+                id
+                step_id
+                service_id
+                build_id
+                repository_id
+                data
+                -- "decodedLogs"
+                ""
+                -- "size"
+                (Bytes.Encode.getStringWidth data)
```

Comment:
I had to modify the log decoder syntax so that we could use the value for `data` to supply the value for `size`.